### ISSUE TITLE
Documentation image partial download causing error

### ISSUE THREAD

**Issue by practicalli-john** [oxsecurity/megalinter#2647]
Is there an issue with the current stable version of the documentation flavor?

Or is this an issue with my local docker setup?

#### Description
I upgraded to the latest MegaLinter documentation flavor a few weeks ago but the download failed due to lack of local disk space.  

After cleaning up disk space it seems one of the overlays is not extracting, even though it breifly reports that overlay is downloading.

The megalinter runner is run using the following command: 
```
npx mega-linter-runner --flavor documentation
```

If I run the command with the `--release beta` version then MegaLinter runner will run without issue.

I haven't found a way to clean the docker cache to remove the overlays as yet and as the stable version is not extracting it is not showing as a docker image that can be removed.

Hunting on stack overflow and other Internet sources has not provided any solutions that have helped so far. 

Node.js version: v18.16.0
Ubuntu (Regolith) Linux 22.04

![image](https://github.com/oxsecurity/megalinter/assets/250870/09e94575-f8bd-4b58-8740-cc54ea555bfb)

**Comment by practicalli-john** [comment]
Apologies, this is not a MegaLinter issue.

I purged everything docker from the operating system and reinstalled.  

The stable version of the documentation flavor works flawlessly

I deleted all Ubuntu packages, `~/.docker` and `/var/lib/docker`

I suspect it was something in the very large docker.raw file in .docker/desktop/VM/0/

The usual docker approach of purging the images and system did not resolve the issue.
